Integrations/EDL/EDL.py

```diff
@@ -76,25 +84,49 @@ def save_context(now: datetime, out_dict: dict):
     demisto.setIntegrationContext(out_dict)
 
 
-def find_indicators_to_limit(indicator_query: str, limit: int) -> list:
+def find_indicators_to_limit(indicator_query: str, limit: int,
+                             panos_compatible: bool = True, url_ps: bool = False) -> list:
     """
```

Comment:
please add args to docstring
why the default value is `False` ?

Reply:
Because this is an 'opt-in' type of option. The default behavior isn't to strip ports from URLs.